packages/amqp/lib/AbstractAmqpPublisher.ts

```diff
@@ -42,6 +42,14 @@ export abstract class AbstractAmqpPublisher<MessagePayloadType extends object>
     }
     resolveSchemaResult.result.parse(message)
 
+    // if the message doesn't have a timestamp field -> add it
```

Comment:
might be useful to explain that this is needed to prevent infinite loops

Reply:
Will add it, thanks Igor 🙇

Reply:
Also, I will update the doc at the end explaining the reason of the new parameters